data/companies/alma/property_data.yaml

```diff
@@ -0,0 +1,90 @@
+
+---
+type: publisher
+company: Alma Media 
+properties:
+  - template: generic
+    channel: display
+    identifier: iltalehti.fi
+    facts:
```

Comment:
You don't need to pull the similar web data - we have a script that does this automatically

Reply:
In fact I don't think you need the property data at all